plugins/inventory/foreman.py

```diff
@@ -108,39 +108,57 @@
         type: int
         default: 600
       want_organization:
-        description: Toggle, if true the inventory will fetch organization the host belongs to and create groupings for the same.
+        description:
+          - Toggle, if true the inventory will fetch organization the host belongs to and create groupings for the same.
+          - Only for Reporting API based inventories.
```

Comment:
If a native english speaker can come up with a better wording, I'll gladly take it ;-)

Reply:
> Only applies to inventories using the Reporting API - attribute is ignored otherwise.

> Exclusive to inventories generated by the Reporting API.

Some combination of the above? Though "Only for Reporting API based inventories" is fairly understandable, the only confusion I'd see would come from not being aware of what that is. Maybe a statement in the Synopsis that explains the different APIs the plugin will use when gathering host information?

Reply:
have a look now, is the description more clear?